dcm4che-core/src/main/java/org/dcm4che3/io/BulkDataDescriptor.java

```diff
@@ -78,31 +79,19 @@ public boolean isBulkData(String privateCreator, int tag, VR vr, int length,
     public final static BulkDataDescriptor PIXELDATA = new BulkDataDescriptor() {
 
         @Override
-        public boolean isBulkData(String privateCreator, int tag, VR vr, int length,
-                                      ItemPointer... itemPointer) {
+        public boolean isBulkData(List<ItemPointer> itemPointer, String privateCreator, int tag, VR vr, int length) {
             if (tag == Tag.PixelData)
                 return true;
-            // Don't need any binary tags larger than a LUT containing 64k elements, which is 64*1024*2
-            switch (vr) {
-            case OB:
-            case OD:
-            case OF:
-            case OL:
-            case OW:
-            case UN:
-                return length > 64*1024*2;
-                default:
-            }
-            return false;
+            // Don't need any private tags larger than 64k
```

Comment:
I was merging stuff from master and this got included - it doesn't make a huge difference but makes things a bit more consistent.

Reply:
This is not part of the master branch! I think it is only part of your pull request, and Gunter will probably not accept this. I think the PIXELDATA BulkDataDescriptor should only remove PixelData. But of course we can always add an additional descriptor, e.g. something like PIXELDATA_AND_LARGE_TAGS.